app/views/settings/login_activities/_login_activity.html.haml

```diff
@@ -0,0 +1,17 @@
+- method_str = content_tag(:span, login_activity.omniauth? ? t(login_activity.provider, scope: 'auth.providers') : t(login_activity.authentication_method, scope: 'login_activities.authentication_methods'), class: 'target')
+- ip_str = content_tag(:span, login_activity.ip, class: 'target')
+- browser_str = content_tag(:span, t('sessions.description', browser: t("sessions.browsers.#{login_activity.browser}", default: "#{login_activity.browser}"), platform: t("sessions.platforms.#{login_activity.platform}", default: "#{login_activity.platform}")), class: 'target')
```

Comment:
I want to see full user-agent in `title` attribute, like active sessions list:
![image](https://user-images.githubusercontent.com/6533808/122323983-fb9c4800-cf62-11eb-953a-b73cdfe74fe7.png)